scripts/control_bar.js

```diff
@@ -78,7 +153,9 @@ var makeImageControl = function(imageElement) {
             $("<span>", {class: "nowrap"}).append(percentInput, "%"),
             " ",
             zoomIn,
-            zoomOut
+            zoomOut,
+            clockRotateInput,
```

Comment:
Consider putting in a spacer (`" "`) so there's some room between the zoom and the rotation control pairs.

Reply:
Done this.